README.md

```diff
@@ -35,6 +36,36 @@ end
 
 Then run `bundle exec rake test`
 
+### Build failure criteria
+
+By default, `ra11y` will cause a build to fail if any errors or warnings are generated. For some users, this might be too strict. `pa11y` allows you to customize the failure criteria with a `Hash`:
+
+```ruby
+options = {
+  failure: {
+    warnings: 10,
+    errors: 10
+  }
+}
+Ra11y::Site.new("./_site", options: options).run
+```
+
+You can also tell `ra11y` only to provide information, but not to affect the build status one way or the other:
+
+```ruby
+options = {
+  inform_only: true
+}
+Ra11y::Site.new("./_site", options: options).run
```

Comment:
Do we need the `options: `? Perhaps the 2nd arg could just be the options hash? This would also mirror HTML proofer.